esp/src/eclwatch/nls/hpcc.js

```diff
@@ -859,6 +860,7 @@ define({root:
     YouAreAboutToDeleteThisQueryset: "You are about to delete this query set. Are you sure you want to do this?",
     YouAreAboutToDeleteThisWorkunit: "You are about to delete this workunit. Are you sure you want to do this?",
     YourBrowserMayNotSupport: "Your browser may not support file(s) of this size",
+    YourScreenWasLocked: "Your screen was locked due to an IP change. Please re-fetch your data, as it may be stale.",
```

Comment:
I think I would be a little more vague - "Your screen was locked by ESP.  Please..."